zulipterminal/ui_tools/views.py

```diff
@@ -950,14 +950,29 @@ def keypress(self, size: urwid_Size, key: str) -> str:
 
 class StreamInfoView(PopUpView):
     def __init__(self, controller: Any, color: str,
-                 desc: str, title: str) -> None:
+                 desc: str, stream_id: int, title: str) -> None:
         # Add 4 (for 2 Unicode characters on either side) to the popup title
         # length to make sure that the title gets displayed even when the
         # content is shorter than the title length (+4 Unicode characters).
-        width = max(len(desc) + 2, len(title) + 4)
-        stream_info_content = [urwid.Text(desc, align='center')]
-        super().__init__(controller, stream_info_content, 'STREAM_DESC', width,
-                         title)
+        # 20 is the length required to show stream settings labels.
+        width = max(len(desc) + 2, len(title) + 4, 20)
+        self.controller = controller
+        self.stream_id = stream_id
+        muted_setting = urwid.CheckBox(label="Mute stream",
```

Comment:
I know this is the label used in the webapp, but it does read a bit strangely to me for some reason. Consistency with the webapp may be important, but this seems like an 'action' rather than a status. Maybe something to think about for a v2, or discuss with frontend.

One idea would be 'muted', which leaves out 'Stream', but perhaps we want a 'Title' above the checkboxes in any case, ie. something like 'Stream settings'?

Reply:
Yes, a status message would be more appropriate in case of a checkbox.